### PR TITLE
LPS-157236 Predefined Value documentation information card can not be closed by user

### PR DESCRIPTION
**Pull request by daiazius**
Related issue: https://issues.liferay.com/browse/LPS-157236

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx [hunk 1]
```diff
@@ -306,23 +308,28 @@ export default function ActionBuilder({
...
-					className="alert-link"
-					href="https://learn.liferay.com/dxp/latest/en/building-applications/objects/creating-and-managing-objects/expression-builder-validations-reference.html"
-					target="_blank"
+			{showAlert && (
+				<ClayAlert
+					className="lfr-objects__side-panel-content-container"
+					displayType="info"
+					onClose={() => {
+						setShowAlert(false);
+					}}
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx:317]: Hey @daiazius, just a simple thing, as you are returning only one function inside onClose, you don't need to put {} around it, like:

```ts
onClose={() => setShowAlert(false)}
```

**Review by danvb112 — CHANGES_REQUESTED** [review]
Good job on this ticket @daiazius, I have just one simple suggestion about it  :wink:

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Pull request closed without merging**